src/server/scripts/Commands/cs_gobject.cpp

```diff
@@ -373,7 +363,7 @@ class gobject_commandscript : public CommandScript
             Unit* owner = ObjectAccessor::GetUnit(*handler->GetSession()->GetPlayer(), ownerGuid);
             if (!owner || !ownerGuid.IsPlayer())
             {
-                handler->PSendSysMessage(LANG_COMMAND_DELOBJREFERCREATURE, ownerGuid.ToString().c_str(), object->GetGUID().ToString().c_str());
+                handler->PSendSysMessage(LANG_COMMAND_DELOBJREFERCREATURE, ownerGuid.GetCounter(), guidLow);
```

Comment:
bug here (both in existing code and proposed)
db entry 274
>Game Object (GUID: %u) is referenced by nonexistent creature %u in GO list, can't be deleted.

counter/lowguid is uint64, %u is not sufficient
always prefer printing guid using ToString(), not its low part